<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>598</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>160</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>160</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic_car_0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic_car_1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic_air">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic_target">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic_prop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>pic</string>
    </property>
   </widget>
   <widget class="QLabel" name="HP_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>456</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HP_G">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>456</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 220, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TIME:180s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change_1">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>510</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ModHP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hpchange_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change_0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ModHP</string>
    </property>
   </widget>
   <widget class="QLabel" name="air_status">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/heart</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hpchange_1">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>510</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="color_of_G">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>360</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/map_black</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Air_A_R">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/heart</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="color_of_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/map_white</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Att_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/verified</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Air_A_G">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>190</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/heart</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Air_con_G">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/plane</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="O_range_G">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>320</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/location_green</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Att_G">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/verified</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="O_range_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/location_red</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Air_con_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/image/plane</pixmap>
    </property>
   </widget>
   <zorder>Air_A_R</zorder>
   <zorder>color_of_R</zorder>
   <zorder>Att_R</zorder>
   <zorder>Air_A_G</zorder>
   <zorder>Air_con_G</zorder>
   <zorder>Att_G</zorder>
   <zorder>color_of_G</zorder>
   <zorder>pic</zorder>
   <zorder>HP_R</zorder>
   <zorder>HP_G</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>pushButton_reset</zorder>
   <zorder>pic_car_0</zorder>
   <zorder>pic_car_1</zorder>
   <zorder>pic_air</zorder>
   <zorder>pic_target</zorder>
   <zorder>pic_prop</zorder>
   <zorder>Status</zorder>
   <zorder>pushButton_change_1</zorder>
   <zorder>lineEdit_hpchange_0</zorder>
   <zorder>pushButton_change_0</zorder>
   <zorder>lineEdit_hpchange_1</zorder>
   <zorder>O_range_R</zorder>
   <zorder>Air_con_R</zorder>
   <zorder>air_status</zorder>
   <zorder>time</zorder>
   <zorder>O_range_G</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
